<?xml version="1.0"?>
<launch>

<node pkg="waypoint_following" type="waypoint_following_node" name="waypoint_following" required="true" output="screen"/>

</launch>
